mailpoet/tests/integration/Newsletter/Blocks/AbandonedCartContentTest.php

```diff
@@ -78,6 +78,14 @@ public function _before() {
       $this->wp->wpDeletePost((int)$product->ID);
     }
 
+
+    if (class_exists('\WC_Post_Types')) {
+      \WC_Post_Types::register_post_types();
```

Comment:
I think we don't have WC in integration tests active at the moment so I'd remove the IF part and leave just the `else` branch now. What do you think?

Reply:
Having it here doesn't do any harms, and for now it also implies that we're better off having WC active but it may create benefits in future, I guess some of the flakey tests come from not using the correct functions to register stuff like this, this is only a guess though

Reply:
@samnajian Right, but on the other hand, I don't think it's a good practice to have different possible flows in tests (if/else based on some circumstances). Again, WC is currently not there so it doesn't make sense to have an IF "in case it is there". Also (noticing just now), the post types should be removed in `cleanup`.

Reply:
I see your point, fine removing the other branch :)